src/utils/assign.js

```diff
@@ -0,0 +1,13 @@
+export default function assign(obj, ...sources) {
```

Comment:
> We can revert to native Object.assign, as the code was already peppered with calls to it before this PR, though.

Yeah, let's revert to using `Object.assign`. This lib already expects it. We should just document all the polyfills expected: Object.assign, ES6 promises, fetch